build.cmd

```diff
@@ -478,12 +478,11 @@ if '%BUILD_PROTO_WITH_CORECLR_LKG%' == '0' (
 rem Build Proto
 if '%BUILD_PROTO%' == '1' (
   if '%BUILD_PROTO_WITH_CORECLR_LKG%' == '1' (
-
-    pushd .\lkg & %_dotnetexe% restore &popd
-    @if ERRORLEVEL 1 echo Error: dotnet restore failed  && goto :failure
-
-    pushd .\lkg & %_dotnetexe% publish project.json -o %~dp0\Tools\lkg -r win7-x64 &popd
-    @if ERRORLEVEL 1 echo Error: dotnet publish failed  && goto :failure
+echo on
+    pushd .\lkg\fsc & %_dotnetexe% restore & popd & if ERRORLEVEL 1 echo Error:%errorlevel% dotnet restore failed & goto :failure
```

Comment:
@brettfo 
Yeah, I'm pretty sure that I thought that too.  It turns out that popd doesn't alter the errorvalue.

A test script
````
pushd c:\temp
cd
c:\kevinransom\visualfsharp\tools\dotnetcli\dotnet restore project.json
popd
if ERRORLEVEL 1 echo Error: dotnet publish failed  && goto :failure 
cd
````

yields this result
````
c:\kevinransom\visualfsharp>pushd c:\temp

c:\temp>cd
c:\temp

c:\temp>c:\kevinransom\visualfsharp\tools\dotnetcli\dotnet restore project.json
error: project.json

c:\temp>popd

c:\kevinransom\visualfsharp>if ERRORLEVEL 1 echo Error: dotnet publish failed    && goto :failure
Error: dotnet publish failed
The system cannot find the batch label specified - failure

c:\kevinransom\visualfsharp>
````